src/Craned/Supervisor/SupervisorServer.cpp

```
@@ -0,0 +1,94 @@
+grpc::Status Supervisor::SupervisorServiceImpl::TerminateTask(
+    grpc::ServerContext* context,
+    const crane::grpc::supervisor::TerminateTaskRequest* request,
+    crane::grpc::supervisor::TerminateTaskReply* response) {
+  g_task_mgr->TerminateTaskAsync(request->mark_orphaned(),
+                                 request->terminated_by_user());
+  response->set_ok(true);
+  return Status::OK;
+}
+
+Supervisor::SupervisorServer::SupervisorServer() {
+  m_service_impl_ = std::make_unique<SupervisorServiceImpl>();
```

Cppcheck (2.10-2)

[performance] Variable 'm_service_impl_' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)